app/src/main/res/values/strings.xml

```diff
@@ -137,5 +137,11 @@
     <string name="title_confirm_delete_timeline">Are you sure to delete this post ?</string>
 
     <string name="action_edit">Edit</string>
+    <string name="action_change_password">Change Password</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.action_change_password` appears to be unused
 Unused resources make applications larger and slow down builds.